rllib/agents/dqn/dqn.py

```diff
@@ -109,41 +103,50 @@
     # to increase if your environment is particularly slow to sample, or if
     # you"re using the Async or Ape-X optimizers.
     "num_workers": 0,
-    # Whether to use a distribution of epsilons across workers for exploration.
-    "per_worker_exploration": False,
     # Whether to compute priorities on workers.
     "worker_side_prioritization": False,
     # Prevent iterations from going lower than this time span
     "min_iter_time_s": 1,
+
+    # DEPRECATED VALUES (set to -1 to indicate they have not been overwritten
+    # by user's config). If we don't set them here, we will get an error
+    # from the config-key checker.
+    "schedule_max_timesteps": -1,
+    "exploration_final_eps": -1,
+    "exploration_fraction": -1,
+    "beta_annealing_fraction": -1,
+    "per_worker_exploration": -1,
 })
 # __sphinx_doc_end__
 # yapf: enable
 
 
-def make_optimizer(workers, config):
+def make_policy_optimizer(workers, config):
+    """
+    Returns:
+        SyncReplayOptimizer: Used for generic off-policy Trainers.
+    """
     return SyncReplayOptimizer(
         workers,
+        # TODO(sven): Move all PR-beta decays into Schedule components.
         learning_starts=config["learning_starts"],
         buffer_size=config["buffer_size"],
         prioritized_replay=config["prioritized_replay"],
         prioritized_replay_alpha=config["prioritized_replay_alpha"],
         prioritized_replay_beta=config["prioritized_replay_beta"],
-        schedule_max_timesteps=config["schedule_max_timesteps"],
-        beta_annealing_fraction=config["beta_annealing_fraction"],
+        prioritized_replay_beta_annealing_timesteps=config[
+            "prioritized_replay_beta_annealing_timesteps"],
         final_prioritized_replay_beta=config["final_prioritized_replay_beta"],
         prioritized_replay_eps=config["prioritized_replay_eps"],
         train_batch_size=config["train_batch_size"],
-        sample_batch_size=config["sample_batch_size"],
         **config["optimizer"])
 
 
-def check_config_and_setup_param_noise(config):
-    """Update the config based on settings.
-
-    Rewrites sample_batch_size to take into account n_step truncation, and also
-    adds the necessary callbacks to support parameter space noise exploration.
+def validate_config_and_setup_param_noise(config):
+    """
```

Comment:
```suggestion
    """Checks and updates the config based on settings.
```

Please follow the pydoc format with a leading sentence (as before).

Reply:
All fixed.